cookbooks/cdo-repository/recipes/default.rb

```diff
@@ -3,44 +3,35 @@
 # Recipe:: default
 #
 
-include_recipe "cdo-github-access"
-
-template "/home/#{node[:current_user]}/.gemrc" do
-  source 'gemrc.erb'
-  user node[:current_user]
-  group node[:current_user]
+# Sync repo via SSH if key is provided.
+include_recipe 'cdo-github-access'
+has_ssh_key = node['cdo-github-access'] && node['cdo-github-access']['id_rsa'] != ''
```

Comment:
What is the desired behavior if node['cdo-github-access']['id_rsa']  is nil/missing?